ethportal-api/src/state.rs

```diff
@@ -92,14 +92,26 @@ pub trait StateNetworkApi {
     ) -> RpcResult<TraceGossipInfo>;
 
     /// Send an OFFER request with given ContentKey, to the designated peer and wait for a response.
+    /// Does not store the content locally.
     /// Returns the content keys bitlist upon successful content transmission or empty bitlist
     /// receive.
     #[method(name = "stateOffer")]
     async fn offer(
         &self,
         enr: Enr,
         content_key: StateContentKey,
-        content_value: Option<StateContentValue>,
+        content_value: StateContentValue,
+    ) -> RpcResult<AcceptInfo>;
+
+    /// Send an OFFER request with given ContentKeys, to the designated peer and wait for a
+    /// response. Requires the content keys to be stored locally.
+    /// Returns the content keys bitlist upon successful content transmission or empty bitlist
+    /// receive.
+    #[method(name = "stateWireOffer")]
+    async fn wire_offer(
```

Comment:
This doesn't make sense for the state network, because content value that is stored can't be validated by receiving side (it's missing proof data). So please remove it.

Alternatively, API can be something like:
`(&self, end: Enr, content: Vec<(StateContentKey, StateContentValue)>)`
which is vectorized version of `offer` above.